dev/io.openliberty.netty.internal/src/io/openliberty/netty/internal/ConfigConstants.java

```diff
@@ -14,9 +14,11 @@
 
 public interface ConfigConstants {
 
-    public AttributeKey<String> NameKey = AttributeKey.valueOf("Name");
-    public AttributeKey<String> HostKey = AttributeKey.valueOf("Host");
-    public AttributeKey<String> ExternalHostKey = AttributeKey.valueOf("ExternalHost");
-    public AttributeKey<Integer> PortKey = AttributeKey.valueOf("Port");
+    public final String ExternalName = "ExternalName";
```

Comment:
Consider String Constant convention EXTERNAL_NAME

Reply:
Sounds good with me, I'll make this change